traits/util/trait_documenter.py

```diff
@@ -129,32 +145,65 @@ def add_directive_header(self, sig):
 
         self.add_line("   :annotation: = {0}".format(definition), "<autodoc>")
 
-    # Private Interface #####################################################
 
-    def _get_trait_definition(self):
-        """ Retrieve the Trait attribute definition
-        """
+def trait_definition(*, cls, trait_name):
+    """ Retrieve the portion of the source defining a Trait attribute.
+
+    For example, given a class::
+
+        class MyModel(HasStrictTraits)
+            foo = List(Int, [1, 2, 3])
+
+    ``trait_definition(MyModel, "foo")`` returns ``"List(Int, [1, 2, 3])"``.
```

Comment:
nit : the function is keyword-only so the doc string needs to be updated.

Reply:
Yep. Will fix.

Reply:
Fixed.